cmd/agent/health.go

```diff
@@ -46,7 +47,7 @@ func handleHeartbeat(w http.ResponseWriter, r *http.Request) {
 func handleVersion(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(200)
 	w.Header().Add("Content-Type", "text/json")
-	json.NewEncoder(w).Encode(versionResp{
+	_ = json.NewEncoder(w).Encode(versionResp{
```

Comment:
Should we handle this error?

Reply:
No since the struct and types are static, so there cant be an error